app/src/main/res/layout/movie_list_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
If it only contains one view, any reason to use `LinearLayout` and not `FrameLayout`?
`FrameLayout` is cheaper in terms of initialization and performance.